opa-iptables/main.go

```diff
@@ -1,9 +1,43 @@
 package main
 
 import (
-	"github.com/open-policy-agent/contrib/opa-iptables/cmd"
+	"flag"
+
+	"github.com/open-policy-agent/contrib/opa-iptables/pkg/logging"
+	"github.com/sirupsen/logrus"
 )
 
+var logger *logrus.Logger
+
 func main() {
-	cmd.Execute()
+	OpaEndpoint := flag.String("opa-endpoint","http://127.0.0.1:8181","endpoint of opa in form of ip:port i.e. 192.33.0.1:8181")
```

Comment:
We should provide defaults or all these 3 parameters. make it easy to run:

http:://127.0.0.1:8181
json-pretty
info